packages/typespec-test/test/anomalyDetector/generated/typespec-ts/src/models/models.ts

```diff
@@ -1071,9 +1071,6 @@ export function univariateUnivariateChangePointDetectionResultDeserializer(
   };
 }
 
-/** Type of APIVersion */
-export type APIVersion = "v1.1";
```

Comment:
Checking with tcgc.

Reply:
Old version:
![{9DC9E9B1-012B-4BEE-9BF2-B86014ABE14A}](https://github.com/user-attachments/assets/88a1f92d-5deb-4510-bd18-e5371f5d983c)
New version:
![{718556C8-FD3E-4B06-A475-21E756383965}](https://github.com/user-attachments/assets/3d0a8c2b-dba2-4459-a66e-22be53fcfa00)

@tadelesh is this expected? https://github.com/v-jiaodi/autorest.typescript/blob/upgrade-116/packages/typespec-test/test/anomalyDetector/spec/main.tsp#L25

Reply:
yes. since `@knownValues` has been deprecated for a long time, we remove the support from tcgc.